JASP-Engine/JASP/R/LSbinomialcommon.R

```diff
@@ -805,6 +863,202 @@
   dat       <- data.frame(x = prior[["parPoint"]], y_start = 0, y_end = 1, g = "Prior = Posterior")
   return(dat)
 }
+.dataPointEstimateBinomial  <- function(data, prior, N, type = c("parameter", "prediction"), estimate = c("mean", "median", "mode"), prop = FALSE){
```

Comment:
To stress the action I recommend renaming this as  .estimateDataPointBinomial

spacing

Reply:
changed